frontend/src/components/chat-interface.tsx

```diff
@@ -175,16 +175,32 @@ export function ChatInterface() {
           curAgentState === AgentState.FINISHED) && (
           <div className="flex flex-col gap-2 mb-2">
             {gitHubToken ? (
-              <SuggestionItem
-                suggestion={{
-                  label: "Push to GitHub",
-                  value:
-                    "Please push the changes to GitHub and open a pull request.",
-                }}
-                onClick={(value) => {
-                  handleSendMessage(value, []);
-                }}
-              />
+              <div className="flex flex-row gap-2">
+                <div className="flex-1">
+                  <SuggestionItem
+                    suggestion={{
+                      label: "Push to GitHub",
+                      value:
+                        "Please push the changes to a remote branch on GitHub, but do NOT create a pull request.",
```

Comment:
Can we update the microagent instructions to send the user a compare link?

e.g.

```
If the user asks you to push your changes, but not create a PR, send them a message with a link to compare the files on the remote branch against the main branch.
```